src/Native/Runtime/unix/unixasmmacros.inc

```diff
@@ -3,6 +3,8 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 //
 
+#include <AsmOffsets.inc>         // generated by the build from AsmOffsets.cpp
```

Comment:
I followed the pattern of the Windows AsmMacros.inc, which imports AsmOffsets.inc for consistency. I can change this though if you prefer.